### DISCUSSION TITLE
Support for Wildchests

### DISCUSSION THREAD

**Discussion by dan28000** [OmerBenGera/WildStacker#488] [Pending Feature Requests] [answered]
It would be nice to add support for wildchests. Specialy needed for storage unit where you can have crap ton of items and if you break the chest then you can crash the server.
There could be 3 modes create one massive stack amount of items being droped from chest or just normal configured stack size in wildstacker config and last mode disabled.

**Comment by OmerBenGera** [comment, marked as answer]
Already a thing. If you have WildStacker installed, WildChests will drop items as stacked.
